src/Microsoft.ApplicationInsights.AspNetCore/project.json

```diff
@@ -93,9 +93,9 @@
         "net451": {
             "dependencies": {
                 "System.Net.Http": "4.3.1",
-                "Microsoft.ApplicationInsights.DependencyCollector": "2.3.0-beta3",
-                "Microsoft.ApplicationInsights.PerfCounterCollector": "2.3.0-beta3",
-                "Microsoft.ApplicationInsights.WindowsServer.TelemetryChannel": "2.3.0-beta3"
+                "Microsoft.ApplicationInsights.DependencyCollector": "2.4.0-beta1",
+                "Microsoft.ApplicationInsights.PerfCounterCollector": "2.4.0-beta1",
+                "Microsoft.ApplicationInsights.WindowsServer.TelemetryChannel": "2.4.0-beta1"
             }
         },
         "netstandard1.6": {
```

Comment:
The NETStandard.Library needs to be upgrade to 1.6.1 (and test projects) if referencing AI base 2.4